src/app/guards/no-auth.guard.spec.ts

```diff
@@ -1,40 +1,45 @@
+import { localStorageSpy } from './../helpers/spies';
+import { LocalStorageService } from 'src/app/services/local-storage.service';
 import { TestBed, inject } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
 import { NoAuthGuard } from './no-auth.guard';
 import { AuthService } from '../services/auth.service';
```

Comment:
Please make this absolute too

Reply:
Thanks, @patrickf949, I have worked on this